yarn.lock

```diff
@@ -812,6 +822,54 @@
     "@smithy/util-waiter" "^2.0.5"
     tslib "^2.5.0"
 
+"@aws-sdk/client-location@3.621.0":
```

Comment:
do we know why this gets added but it wasn't before?

Reply:
I'm not sure why it wasn't there already, afaik it should be in the yarn lock because `@aws-amplify/geo` is a dev dependency of ui-react-geo